docs/layouts/shortcodes/generated/forst_configuration.html

```diff
@@ -0,0 +1,120 @@
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-write</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Whether to let write requests executed within the coordinator thread.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.read-io-parallelism</h5></td>
+            <td style="word-wrap: break-word;">3</td>
+            <td>Integer</td>
+            <td>The number of threads used for read IO operations in the executor.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.write-io-parallelism</h5></td>
+            <td style="word-wrap: break-word;">1</td>
+            <td>Integer</td>
+            <td>The number of threads used for write IO operations in the executor. Only valid when 'state.backend.forst.executor.inline-write' is false.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.local-dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The local directory (on the TaskManager) where ForSt puts some meta files. Per default, it will be &lt;WORKING_DIR&gt;/tmp. See <code class="highlighter-rouge">process.taskmanager.working-dir</code> for more details.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.memory.fixed-per-slot</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>MemorySize</td>
+            <td>The fixed total amount of memory, shared among all ForSt instances per slot. This option overrides the 'state.backend.forst.memory.managed' option when configured.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.memory.fixed-per-tm</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>MemorySize</td>
+            <td>The fixed total amount of memory, shared among all ForSt instances per Task Manager (cluster-level option). This option only takes effect if neither 'state.backend.forst.memory.managed' nor 'state.backend.forst.memory.fixed-per-slot' are not configured. If none is configured then each ForSt column family state has its own memory caches (as controlled by the column family options). The relevant options for the shared resources (e.g. write-buffer-ratio) can be set on the same level (config.yaml).Note, that this feature breaks resource isolation between the slots</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.memory.high-prio-pool-ratio</h5></td>
+            <td style="word-wrap: break-word;">0.1</td>
+            <td>Double</td>
+            <td>The fraction of cache memory that is reserved for high-priority data like index, filter, and compression dictionary blocks. This option only has an effect when 'state.backend.forst.memory.managed' or 'state.backend.forst.memory.fixed-per-slot' are configured.</td>
```

Comment:
I assume the valid values are between 0 and 1. what does 0 and 1 mean? Should 0 and 1 not be used?

Reply:
You shouldn't use 0 or 1, since for any type of memory usage you need to reserve some space.